### ISSUE TITLE
I can't get expected results when training mask-rcnn

### ISSUE THREAD

**Issue by QY1996** [facebookresearch/maskrcnn-benchmark#669]
## ❓ Questions and Help
I tried to train mask-rcnn using COCO 2014 dataset and the configure file is e2e_mask_rcnn_R_50_FPN_1x.yaml. I changed the yaml file a little to train on a single GPU, the yaml file is like this:
![image](https://user-images.githubusercontent.com/49559560/56044856-aa84de00-5d72-11e9-8ce9-ae50f883ed8b.png)
![image](https://user-images.githubusercontent.com/49559560/56044908-c1c3cb80-5d72-11e9-9cbc-9b2261e8e7a5.png)
I tested my dataset using pycocotools, it looks like that, seems all right. I also checked the bounding box and it's correct organized by the "xywh" mode.
![image](https://user-images.githubusercontent.com/49559560/56045488-e9676380-5d73-11e9-8fdf-afe79e6c6452.png)
    The problem is when I'm training, the loss first drops and when it gets to about 0.3, it will not decrease any more until the training is over. When I test the model I trained, it outputs the wrong result that makes no sense.(Sorry I deleted the result image, but the bounding box is everywhere and the category is wrong and the probability is very low like about 0.05... Anyway it's obviously a wrong result) By the way, I also tried to adjust the learning rate but it didn't work.

I also tried to load a pre-trained model from 

> catalog://Caffe2Detectron/COCO/35858933/e2e_mask_rcnn_R-50-FPN_1x

When I do test, it works well. But when I use this model to finetune, the loss is about 0.7 and won't decrease. And the model's performance is getting worse along the training process.

![image](https://user-images.githubusercontent.com/49559560/56047662-80ceb580-5d78-11e9-8fe5-87a3d623df9e.png)

I need to know what might be the reason of this issue.

# Here is some of my environment:
torch-nightly                     1.1.0.dev20190410 
torchvision-nightly               0.2.3   
CUDA                                 10.0
GPU                                    NVIDIA 2080 Ti

Thanks for anyone who answers my question!

**Comment by fmassa** [comment]
You have changed the batch size but you didn't adapt properly the number of iterations nor the learning rate of your model.

This explains the difference. You should follow the instructions in https://github.com/facebookresearch/maskrcnn-benchmark#single-gpu-training
